packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/App.kt

```diff
@@ -36,10 +37,21 @@ interface App {
 
     companion object {
         /**
-         * TODO
+         * Create an [App] with default settings.
+         * @return
          */
-        fun create(
-            configuration: AppConfiguration,
-        ): App = AppImpl(configuration as AppConfigurationImpl)
+        fun create(appId: String): App {
+            Validation.checkEmpty(appId, "appId")
+            return create(AppConfiguration.Builder(appId).build())
+        }
+
+        /**
+         * Create an [App] according to the given [AppConfiguration].
+         *
+         * @param configuration The configuration to use for this [App] instance.
+         * @see AppConfiguration.Builder
+         */
+        fun create(configuration: AppConfiguration): App =
+            AppImpl(configuration as AppConfigurationImpl)
```

Comment:
We don't have test for this. I wouldn't be against leaving it like that as I wouldn't expect that people throw their own `AppConfiguration`-implementation. But maybe do `AppImpl(Validation.checkType<AppConfigurationImpl>(configuration, "configuration))` so we get a better error message.

Reply:
For now, I would probably prefer not to add code like this. Mostly because we would need to add it in a lot of places to be consistent.

Maybe wait to see if people accidentally start passing their own implementations around before doing it?